src/geo_ip_reader.rs

```diff
@@ -323,15 +320,15 @@ where
     /// let record = geo_ip.get_record("185.90.90.120");
     /// println!("Geographical Record: {:?}", record);
     /// ```
-    ///
-    pub fn get_record(&mut self, ip_number: &str) -> HashMap<&str, Option<String>> {
+    pub fn get_record(&mut self, ip_number: &str) -> Record {
         // Get the offset of the country record for the given IP address
         let seek_country = self
             .get_country(ip_to_number(ip_number).try_into().unwrap())
             .unwrap();
+
         // Check if the offset is equal to the total number of database segments
         if seek_country == self.database_segments.try_into().unwrap() {
-            return HashMap::new();
+            todo!("Error handling")
```

Comment:
I think error handling should be a part of same release but other pr (@wiseaidev)